docs/source/includes/alpn_setup.rst

```diff
@@ -0,0 +1,5 @@
+If you want to run a small pipeline on your machine before running it in parallel on Compute Engine, you will need `ALPN`_ since many of these pipelines require it.  When running locally, this must be provided on the boot classpath but when running on Compute Engine Dataflow workers this is already configured for you. You can download the correct version from `here <http://mvnrepository.com/artifact/org.mortbay.jetty.alpn/alpn-boot>`__.  For example::
```

Comment:
I am unclear about the phrase "correct version". Is there something specific implied there that needs elaboration?
Or should this just say "You can download ALPN from `here ...`"?